Sources/MediaCategories/MediaCategoryViewController.swift

```diff
@@ -21,7 +21,7 @@ class VLCMediaCategoryViewController: UICollectionViewController, UICollectionVi
     private var searchController: UISearchController?
     private let searchDataSource = VLCLibrarySearchDisplayDataSource()
     private lazy var editController: VLCEditController = {
-        let editController = VLCEditController(model: self.model)
+        let editController = VLCEditController(services:self.services, model: self.model)
```

Comment:
Do we need all the services here? 

If not we could just pass the one we need.

Additionally, extra `self.`.

Reply:
self is needed because of the lazy and I can just pass in the manager

Reply:
Locally, XCode doesn't seem to complain about the `self.` changes.